app/user.go

```diff
@@ -912,7 +912,7 @@ func (a *App) UpdateActive(c *request.Context, user *model.User, active bool) (*
 			return nil, model.NewAppError("UpdateActive", "app.user.update.finding.app_error", nil, err.Error(), http.StatusInternalServerError)
 		}
 	}
-	ruser := userUpdate.New
+	ruser := userUpdate.New.DeepCopy()
```

Comment:
This seem to solve the issue but, I think returning new user in the old users pointer isn't correct. Not sure if it would be part of this PR anyway. Because, semantically I would expect the new one to have a fresh struct and I probably wouldn't do anything with the old one. WDYT?

Reply:
Agree. Makes sense.